library/core/src/ptr/const_ptr.rs

```diff
@@ -1899,3 +1947,219 @@ impl<T: ?Sized> PartialOrd for *const T {
         *self >= *other
     }
 }
+
+#[cfg(kani)]
+#[unstable(feature = "kani", issue = "none")]
+mod verify {
+    use crate::kani;
+    use core::mem;
+    // Constant for array size used in all tests, for performance reason
```

Comment:
Could you give a bit more info? You're using a `BUF_SIZE` of 200 for other harnesses. How is the verification time for 5 here? Does it blow up if you double it? Is this enough to explore all functions for slices?

Reply:
Hi @szlee118, could you elaborate on this matter?

Reply:
The verification time slightly increase, but it doesn't blow up when doubling it.

`BUF_SIZE`:
- Represents the memory buffer size (in bytes) allocated for testing raw pointer arithmetic.
- Needs to be large enough to ensure pointer operations like add, sub, and offset have enough valid memory space to operate on, covering the full range of test scenarios.

`ARRAY_SIZE`:
- Represents the number of elements in the slice or array being tested.
- Doesn't need to be large because all possible values for the slice are explored, including edge cases like pointers at the start, middle, and end of the slice. Symbolic execution generalizes across all possible elements, regardless of the actual array size.

Reply:
I'd much appreciate if the above comment could be reflected as a comment in the source code. Anyone looking at those constants in future should know why those particular numbers have been chosen.